srcpkgs/wezterm/template

```diff
@@ -0,0 +1,68 @@
+# Template file for 'wezterm'
+pkgname=wezterm
+version=20220905
+revision=1
+_srcver=${version}-102802-7d4b8249
```

Comment:
```suggestion
_srcver=${version}-102802-7d4b8249
archs="x86_64* i686* aarch64* arm*" # ring
```
`ring` is in the dependency tree for this and has issues on ppc so we can restrict it preemtively